src/Neo/Network/P2P/RemoteNode.ProtocolHandler.cs

```diff
@@ -415,12 +411,12 @@ private void OnVersionMessageReceived(VersionPayload payload)
 
         private void OnTimer()
         {
-            DateTime oneMinuteAgo = TimeProvider.Current.UtcNow.AddMinutes(-1);
+            var oneMinuteAgo = TimeProvider.Current.UtcNow.AddMinutes(-1);
```

Comment:
this is still quite empirically calibrated. I remember when this was added.
We need to improve this logic at all.